src/index.js

```diff
@@ -314,5 +314,29 @@ Branch.prototype.lastAttributedTouchData = function lastAttributedTouchData() {
   return execute("lastAttributedTouchData");
 };
 
+Branch.prototype.getBranchQRCode = function getBranchQRCode(
+  qrCodeSettings,
+  branchUniversalObject,
+  analytics,
+  properties
+) {
+  var args = [];
```

Comment:
Does this need to be wrapped in a Promise? There is a network request being made here.
cc @JagadeeshKaricherla-branch